src/sass/components/_header.scss

```diff
@@ -181,19 +182,56 @@ header {
         }
 
         i:last-child {
-          position: absolute;
-          right: 10px;
-          z-index: 0;
+          padding: 10px;
         }
 
-        select {
-          -webkit-appearance: none;
-          border: 0;
-          background-color: transparent;
-          outline: none;
-          padding: 5px 30px 5px 35px;
-          font-size: 14px;
-          position: relative;
+        ul {
+          margin: 0;
+
```

Comment:
What is this empty line for?

Reply:
only God knows... :) I need to be more careful.